Source/multi.cpp

```diff
@@ -818,7 +795,7 @@ bool multi_init_single(GameData *gameData)
 	}
 
 	unused = 0;
-	if (!SNetCreateGame("local", "local", "local", 0, (char *)&sgGameInitInfo, sizeof(sgGameInitInfo), 1, "local", "local", &unused)) {
+	if (!SNetCreateGame("local", "local", (char *)&sgGameInitInfo, sizeof(sgGameInitInfo), &unused)) {

```

Comment:
They really liked using all kinds of BYTE and char and unsigned char pointers instead of void* for 'pointer to unknown type'.

reinterpret_cast

Reply:
Not sure if they did char or void. Can be hard to tell.